android/src/main/java/im/shimo/react/prompt/RNPromptFragment.java

```diff
@@ -201,6 +201,17 @@ public Dialog createDialog(Context activityContext, Bundle arguments) {
 
         mInputText = input;
 
+        alertDialog.setOnShowListener(new DialogInterface.OnShowListener()
+        {
+            @Override
+            public void onShow(final DialogInterface dialog)
+            {
+                input.requestFocus();
+                ((InputMethodManager) alertDialog.getContext().getSystemService(Context.INPUT_METHOD_SERVICE)).showSoftInput(input, 0);
```

Comment:
maybe below code behave better：
InputMethodManager imm = (InputMethodManager) alertDialog.getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
if (imm != null) {
    input.requestFocus();
    imm.showSoftInput(view, 0);
}

Reply:
@wsong910 - are you waiting on me to make this change before accepting this PR? Or are you researching when imm can be null so we can force get imm in that situation to ensure keyboard shows?